src/main/java/com/youngcamp/server/config/SecurityConfig.java

```diff
@@ -41,12 +41,19 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 authorize
                     .requestMatchers("/api/swagger-ui/**", "/v3/api-docs/**")
                     .hasRole("DEV")
+                    .requestMatchers("/api/admin/login")
+                    .permitAll()
+                    .requestMatchers("/api/admin/**")
+                    .authenticated()
```

Comment:
/api/admin/**에 대해서 admin만 가능하게 하려고 의도하신 거라면 
```suggestion
 .requestMatchers("/api/admin/**")
                    .hasRole("ADMIN") 
```
이렇게 수정되어야 해요.

Reply:
어차피 로그인할 수 있는 계정이란 게 Admin밖에 없어서
로그인이 된 사용자 == 관리자 로 인식하고 저렇게 설정을 했었던 것 같네요.

바로 위에 DEV는 hasRole로 잡아줬으니
일관성을 위해서 바꿔주는 게 가독성적으로 좋을 것 같습니다! 👍 
이거로 바꿔 낄게요!